gallery/src/components/NxGlobalSidebar/NxGlobalSidebarPage.scss

```diff
@@ -0,0 +1,9 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+.gallery-custom-sidebar-content * {
```

Comment:
This is kinda quick and dirty.  We should probably have less sloppy code in an example

Reply:
> This is kinda quick and dirty. We should probably have less sloppy code in an example

It was meant to be quick and dirty. I guess it was too quick, too dirty. Cleaner version here: https://github.com/sonatype/sonatype-react-shared-components/pull/371/commits/50db41402d0baec8706a47d1345d6958a1669925